modules/os2forms_organisation_openid_connect/src/EventSubscriber/OrganisationUserIdEventSubscriber.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\os2forms_organisation_openid_connect\EventSubscriber;
+
+use Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException;
+use Drupal\Component\Plugin\Exception\PluginNotFoundException;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\os2forms_organisation\Event\OrganisationUserIdEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Organisation user id event subscriber.
+ */
+class OrganisationUserIdEventSubscriber implements EventSubscriberInterface {
+
+  /**
+   * The current user.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected AccountInterface $currentUser;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected EntityTypeManagerInterface $entityTypeManager;
+
+  /**
+   * The constructor.
+   */
+  public function __construct(AccountInterface $currentUser, EntityTypeManagerInterface $entityTypeManager) {
+    $this->currentUser = $currentUser;
+    $this->entityTypeManager = $entityTypeManager;
+  }
```

Comment:
Use https://stitcher.io/blog/constructor-promotion-in-php-8 for this reduce the amount of boilerplate code.

Reply:
Updated.